### DISCUSSION TITLE
Is it possible to search the old Spectrum threads?

### DISCUSSION THREAD

**Discussion by bigsee** [vercel/vercel#4259] [General] [answered]
...or have they been migrated somewhere? There were some great solutions on there and I often checked back there. Is all of that collective knowledge now lost?

A direct link to an old thread will still work: [https://spectrum.chat/zeit/now/cors-policy-block~9871f3e1-652b-4c63-809d-d5aa53ffc176](https://spectrum.chat/zeit/now/cors-policy-block~9871f3e1-652b-4c63-809d-d5aa53ffc176)

...but I have not found a way to access the search. Anyone got any ideas or do we just have to hope we get lucky with search engine indexing?

**Comment by mcsdevv** [comment, marked as answer]
Hey @bigsee! Whilst there was some useful material on Spectrum, there was significantly more material that was out of date and tripping users up. With GitHub Discussions, we can change marked answers as and when best practices change, and avoid people being misled by outdated information.

If there's anything you're particularly interested in please feel free to create a Discussion about it and we'll be sure to help out where we can.

**Reply by bigsee** [reply]
Thanks for the update! I have to question the wisdom of getting rid of all that knowledge, especially when everything on here is still so sparse and the Spectrum threads often filled gaps in the (content or clarity of the) documentation. Guess we'll have to start from the beginning then. 🤷🏽‍♂️

**Reply by mcsdevv** [reply]
I'm in agreement that there was some useful stuff in there, and that's why we'll be making a knowledge base available very soon!
